### PR TITLE
Fix for Java 8 compatibility - IDETECT-4436

### PR DESCRIPTION
**Pull request by gopannabd**
Fix for Java 8 compatibility.
IDETECT-4436

### CODE REVIEW

**Review by dterrybd — APPROVED** [review]

**Review by psantos1113 — APPROVED** [review]

File: src/main/java/com/synopsys/blackduck/upload/client/uploaders/BinaryUploader.java [hunk 1]
```diff
@@ -62,15 +64,16 @@ public class BinaryUploader extends AbstractUploader<BinaryUploadStatus> {
...
-        );
+        final Map<String, File> contentMap = new HashMap<String, File>() {{
+            put("fileupload", filePath.toFile());
+        }};
+        final Map<String, String> metaDataMap = new HashMap<String, String>() {{
+            put("projectName", binaryScanRequestData.getProjectName());
+            put("version", binaryScanRequestData.getVersion());
+            put("codeLocationName", binaryScanRequestData.getCodeLocationName().orElse(""));
+            put("codeLocationUri", binaryScanRequestData.getCodeLocationUri().orElse(""));
+        }};
```
Comment (ChomickiM) [inline src/main/java/com/synopsys/blackduck/upload/client/uploaders/BinaryUploader.java]: The instance initializers appear to be coming up as a code smell in sonar. Can we split initializing the Maps and a putting the key/values?

Comment (gopannabd) [reply src/main/java/com/synopsys/blackduck/upload/client/uploaders/BinaryUploader.java]: Those 'code smells' are silly.

Sure.

Comment (gopannabd) [reply src/main/java/com/synopsys/blackduck/upload/client/uploaders/BinaryUploader.java]: Done

**Comment by gopannabd** [comment]
@ChomickiM  Will wait for you to finish your review.

**Review by ChomickiM — APPROVED** [review]

**Review by DanaMaxfield — APPROVED** [review]

**Pull request merged**